test/units/TEST-17-UDEV.device_is_processing.sh

```diff
@@ -36,7 +36,7 @@ cat >/run/udev/rules.d/99-testsuite.rules <<EOF
 SUBSYSTEM=="net", ACTION=="change", KERNEL=="${IFNAME}", OPTIONS="log_level=debug", RUN+="/usr/bin/sleep 1000"
 EOF
 
-systemctl restart systemd-udevd.service
+udevadm control --reload
```

Comment:
wouldn't it make sense to use "systemctl reload systemd-udevd" too somewhere, just to make sure this also works correctly?